content/blog/bare-metal-provisioning-on-hpe-greenlake-using-terraform.md

```diff
@@ -0,0 +1,359 @@
+
+![API Client Created](/img/greenlake_conolse_apiclient_created.png "API Client Created")
+
+5. In the API Clients page, select the newly created client, and click on **Create Assignment** button.
+
+
+6. Assign the roles **BMAAS Access Viewer** and **BMAAS Access Project Contributor** on the **Space: Default.**
+
+![BMaaS Roles](/img/greenlake_console_createbmaasassignment.png "BMaaS Roles")
+
+The API Client is now ready to be used to run the Terraform resources.
+
+## Select the Compute Group ID
+
+Compute Group is a logical grouping of bare metal resources (Compute Instances, Networks, SSHKeys, etc.) that a team of Cloud Consumers can consume. You must specify the compute group ID to interact with the bare metal service resources. You can create a new compute group or select the existing one from the HPE GreenLake console. Make a note of the compute group ID because you need it to set a variable.
+
+***Note: Compute Group is AKA Project.***
+
+1. Navigate to HPE GreenLake for Private Cloud Services card -> Bare Metal -> Compute Groups.
+
+![BMaaS Compute Groups](/img/compute_group_list.png "BMaaS Compute Groups")
+
+2. Click on the desired compute group and then extract the ID from the browser URL seen at that time.\
+   \
+   This will be exported in the environment variable **HPE_METAL_PROJECT_ID** in the later section.
+
+![Compute Group ID](/img/compute_group_id.png "Compute Group ID")
+
+ ﻿ 
+
+## Install Terraform
+
+Next, get your system ready to run Terraform. In case this has not been done yet:
+
+1. Download and install Terraform, version v0.13 or later.
+   For more information, see [https://learn.hashicorp.com/tutorials/terraform/install-cli](<1. https://learn.hashicorp.com/tutorials/terraform/install-cli>). 
+2. Verify the installation with **terraform -help.** 
+
+   At this point, you are ready to start building your infrastructure description file. 
+
+# Deploy Compute Instance
+
+## Select Terraform provider with bare metal service configurations
+
+1. #### Export the following environment variables on your setup.
+
+   E﻿xport the Tenant ID:
+
+   ```shell
+   export HPEGL_TENANT_ID="<Tenant ID>
+   ```
+
+   Export the API Client credentials that you obtained when you created an API Client within HPE GreenLake Central:
+
+   ```shell
+   export HPEGL_USER_ID="<API Client ID>"
+   export HPEGL_USER_SECRET="<API Client Secret>"
+   export HPEGL_IAM_SERVICE_URL="<Issuer URL>"
+   ```
+
+   E﻿xport the Compute Group ID:
+
+   ```
+   # compute group/project ID
+   export HPEGL_METAL_PROJECT_ID="<Compute Group ID>"
+   ```
+
+    ﻿Export bare metal service REST URL:
+
+   ```
+   # Production Environment:  https://client.greenlake.hpe.com/api/metal
+   # Integration Environment: https://client.greenlake.hpe-gl-intg.com/api/metal
+   # local development: http://localhost:3002
+
+   export HPEGL_METAL_REST_URL="<Metal Service REST Base URL>"
+   ```
+2. #### Configure the Terraform provider.
```

Comment:
Well, at least this bullet is consistent in terms of formatting with #1 above. Just need to double check to see if you wanted the first section to be similarly formatted.